client/src/components/ProductDetails/ImageGallery.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import './prodStyles.css';
+
+function ImageGallery ({imageGallery}) {
+
+  const [currentImage, updateMainImage] = useState(imageGallery[0])
+
+  const setThumbnailClass = (image) => {
+    return image === currentImage ? "imageThumbnailMain" : "imageThumbnail"
+  }
+  const enterExpandedView = () => {
```

Comment:
Did you mean to leave this function empty?

Reply:
yes. I know I need to implement it, but not sure how just yet. Its what ill be doing tomorrow

Reply:
I see. One suggestion for stubbing out functions is to return null and add a 'todo' comment, so then we know that you're planning to implement that function later on.